aviatrix/resource_aviatrix_saml_endpoint.go

```diff
@@ -129,7 +129,11 @@ func resourceAviatrixSamlEndpointRead(d *schema.ResourceData, meta interface{})
 		d.Set("custom_saml_request_template", saml.MsgTemplate)
 	}
 
-	d.Set("controller_login", saml.ControllerLogin)
```

Comment:
I think this is still boolean

Reply:
SamlEndpoint.ControllerLogin is changed to string for Post.